core/src/main/java/com/graphhopper/routing/AbstractNonCHBidirAlgo.java

```diff
@@ -59,7 +60,8 @@ public AbstractNonCHBidirAlgo(Graph graph, Weighting weighting, TraversalMode tM
         this.flagEncoder = weighting.getFlagEncoder();
         this.graph = graph;
         this.nodeAccess = graph.getNodeAccess();
-        edgeExplorer = graph.createEdgeExplorer();
```

Comment:
Another option would be using `graph.createEdgeExplorer()` 'everywhere' (unfiltered edges) and then filter them by whatever one needs to do. This is maybe the most flexible approach, but it is also a bit more messy to do the filtering 'everywhere'?